pages/customers/fey.mdx

```diff
@@ -0,0 +1,78 @@
+
+Simpler code and developer workflow weren't the only improvements. The new data pipelines completed in a fraction of the time of their previous system: **on average, data was now processed 50x faster**.
+
+Finally, they saw a major reduction in unit costs - **workloads that previously cost them $2,800 monthly to run could now be completed for less than $100** due to significant efficiencies. This improved margins and gives them much more room to scale.
+
+> As soon as companies release their data, we want to have it in Fey. **With our old tool it would take up to a day** for them to show up, which was a result of mostly complicated configuration. And **with Inngest now it's like almost instant**, maximum 30 minutes.
+
+## Combining ingestion with OpenAI
+
+One of the competitive advantages of Fey is how they use generative AI to enhance stock research reporting.
+
+There are two features where Fey employs OpenAI. First is summarizing news articles to provide a “human-friendly” rundown of different types financial data.
+
+![A screenshot of Fey's news page](/assets/customers/fey/fey-news-page.jpeg)
+
+The second feature are SEC filings, both 10-Ks and 10-Qs. Fey generates a summary of the highlights, risks, and strengths. In this way, users don't have to deal with the cognitive load of financial jargon.
+
+![A screenshot of Fey's news page](/assets/customers/fey/fey-news-page-2.png)
+
+Both features use GPT-4 model via their Assistants API. Both require a complex data pipeline. For example, to achieve best results when reporting on SEC filings, each piece needs to be split into a few sections, which then are run through different prompts.
+
+Both features are orchestrated using Inngest's [`step.invoke()`](/docs/reference/functions/step-invoke), which allows you to asynchronously call another function and handle the result, making it perfect for complex workflows.
+
+![Inngest UI used by Fey](/assets/customers/fey/fey-inngest-ui.jpeg)
+
+## Inngest Benefits
+
+The adoption of Inngest resulted in several positive outcomes for Fey:
+
+1. **Concise codebase:** Inngest's SDK is designed with developers in mind, which results in elegant and self-evident code. Moreover, given that all code lives in a single codebase now, there's no need for duplicated logic in different language codebases. 
+2. **Decreased context-switching:** Connected to the previous point, it is easier to introduce new features because all code lives in the same place. Working across frontend and backend is not necessary anymore.
+3. **50x faster execution:** When companies release earnings, Fey needs to process this data and serve it to the users, which with the previous provider could take up to a day. With Inngest, the data is available instantly (or up to 30 minutes).
```

Comment:
suggestion: Felt weird to say instant then caveat with a delay, maybe we should say the new max time then mention that it's often instant?
```suggestion
3. **50x faster execution:** When companies release earnings, Fey needs to process this data and serve it to the users, which with the previous provider could take up to a day. With Inngest, the data is available within 30 minutes or most often, instantly.
```